rpc_configs/substrate.yml

```diff
@@ -60,6 +60,8 @@ methods:
         ty: BlockHash
         optional: true
         inject: true
+    # This will not be rate limited.
+    rate_limit_weight: 0
```

Comment:
this is the default config and I don't want to make it default to no limit

Reply:
Default is 1

Reply:
oh you mean don't chain these configs, ok got it